### PR TITLE
Nollning matrix: rewrites view logic

### PR DESCRIPTION
**Pull request by davidwessman**
Adds description and keywords to Nollning pages

### CODE REVIEW

File: app/controllers/nollning/nollnings_controller.rb [hunk 1]
```diff
@@ -7,7 +7,13 @@ def index
   end
 
   def matrix
-    @events = Event.includes(:translations).slug(:nollning)
+    @dates = Date.new(2016, 8, 22)..Date.new(2016, 9, 23)
```
Comment (navro) [inline app/controllers/nollning/nollnings_controller.rb]: 😢

Comment (davidwessman) [inline app/controllers/nollning/nollnings_controller.rb]: Yep, what should we do?

Comment (navro) [inline app/controllers/nollning/nollnings_controller.rb]: > Yep, what should we do?

I think it's ok in this case, but hard coded values always makes me sad 😉

**Comment by navro** [comment]
+1

File: app/controllers/nollning/nollnings_controller.rb (hunk 1 above)
Comment (davidwessman) [inline app/controllers/nollning/nollnings_controller.rb]: I was thinking of creating a whole `Nollning`-model but decided not to.
However that might actually be wort it - could be cool for history as well.

**Pull request merged**